### PR TITLE
Add license title

### PR DESCRIPTION
**Pull request by waldyrious**
It's not strictly required, but it's useful metadata, and part of the recommended license template text (see http://choosealicense.com/licenses/isc/ and https://opensource.org/licenses/isc-license)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -11,4 +13,3 @@ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
-
```
Comment (schachmat) [inline LICENSE]: Please keep the newline at end of file.

Comment (waldyrious) [inline LICENSE]: I did leave a newline at the end of the file (there were two). I suspect this is a github rendering issue. In fact, if I attempt to edit [the file in my branch](https://github.com/waldyrious/wego/edit/patch-1/LICENSE), it does show up in the editor with 16 lines, the last one being a newline. Can you confirm?

**Comment by schachmat** [comment]
You are right. Merged. Thanks for your contribution.

**Comment by waldyrious** [comment]
Thanks for the review @schachmat.

For future reference (since github didn't connect the commit with this thread), the change was included in ac6c67b.

**Comment by schachmat** [comment]
Yeah, that's probably because I removed the long description from the commit message. Also I never use the "merge" button from the web interface since it creates a non-linear history.

**Comment by waldyrious** [comment]
@schachmat you should give the new [squash and merge feature](https://github.com/blog/2141-squash-your-commits) a try :)

**Comment by schachmat** [comment]
Thanks for the hint.

**Pull request closed without merging**